packages/web/package.json

```diff
@@ -106,6 +106,7 @@
     "bitski": "0.10.9",
     "bn.js": "5.1.0",
     "body-scroll-lock": "4.0.0-beta.0",
+    "bs58": "5.0.0",
```

Comment:
was this not already a dependency? We use it in the common solana service...

Reply:
all uses of it i see are in other packages